packages/hd/src/keystore.ts

```diff
@@ -94,8 +100,8 @@ export default class Keystore {
   // Create an empty keystore object that contains empty private key
   static createEmpty(): Keystore {
     const saltSize = 32;
-    const salt: Buffer = crypto.randomBytes(saltSize);
-    const iv: Buffer = crypto.randomBytes(16);
+    const salt: Buffer = Buffer.from(randomBytes(saltSize));
```

Comment:
Avoid using `Buffer` which is only available in Node.js.

`randomBytes` returns `Uint8Array` which can replace the `Buffer`

Reply:
The plan is to replace `crypto` first, then replace `Buffer` with `Uint8Array` as described in  https://github.com/ckb-js/lumos/pull/645
<img width="940" alt="image" src="https://github.com/ckb-js/lumos/assets/7459812/ff5e6ee2-8ee4-4143-9416-3087410ec3f3">

Reply:
Got it